siddpubs-conf.bib

```diff
@@ -52,6 +52,13 @@ @string{icml
 
 % 2019 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{hanjun2019shear,
```

Comment:
Please use your last name as the key, i.e. `song2019shear`.

Reply:
ok, done!